templates/commands/render_action_regexreplace.go

```diff
@@ -66,24 +66,21 @@ func actionRegexReplace(ctx context.Context, rr *model.RegexReplace, sp *stepPar
 		}
 	}
 
-	for _, p := range rr.Paths {
-		if err := walkAndModify(ctx, p.Pos, sp.fs, sp.scratchDir, p.Val, func(b []byte) ([]byte, error) {
-			for i, rr := range rr.Replacements {
-				cr := compiledRegexes[i]
-				allMatches := cr.FindAllSubmatchIndex(b, -1)
+	if err := walkAndModify(ctx, sp.fs, sp.scratchDir, rr.Paths, func(b []byte) ([]byte, error) {
```

Comment:
Hm this might be a preexisting bug. We should do call `parseAndExecuteGoTmpl` on rr.Paths before passing them to walkAndModify, so things like `paths: ['{{.myinput}}/a.txt}}']` are handled.

Or, maybe we could have walkAndModify do the template expansion on paths, so all the callers don't have to have the same boilerplate?

Reply:
I had noticed that too but originally thoguht it was intentional. I filed a bug #142 for the broken actions as well as a feature request #143  for your suggestion.